settings.py

```diff
@@ -262,9 +266,6 @@
         'http://localhost:3000',
         'http://localhost:8000',
     )
-    # the collectstatic step in docker build runs without env variables set, and uncommenting these lines breaks the docker build
-    # STATICFILES_DIRS.append(STATIC_ROOT)
```

Comment:
previously you needed to uncomment these lines locally in order to use `localhost:8000` with the generate static files. Is this no longer the case?

Reply:
I don't fully follow your question or why this comment was in here in the first place - can we chat about this in person? 

This prob doesn't answer the question but I ran collectstatic on my local with no issues, all of the static files were collected successfully into the static/ directory.

Reply:
discussed offline, no longer relevent since build assets are not in source control